data/net/minecraft/core/BlockPos.mapping

```diff
@@ -86,6 +94,7 @@ CLASS net/minecraft/core/BlockPos
 		ARG 0 pos
 		ARG 2 direction
 	METHOD randomBetweenClosed (Ljava/util/Random;IIIIIII)Ljava/lang/Iterable;
+		COMMENT Creates an Iterable with n Objects in the area.
```

Comment:
This comment doesn't really make sense. Firstly `n` is a bad place holder here, I know it was used in the data we worked off, but back then the param was also called `n`.

Also the capitalisation of `Objects` just adds more confusion.